README.md

```diff
@@ -8,22 +9,39 @@ You need to install the following tools to build locally:
 
 * The [protoc](https://grpc.io/docs/protoc-installation/) compiler
 * The following compiler plugins:
+
 ```bash
-$ go get -u -v google.golang.org/protobuf/cmd/protoc-gen-go \
+go get -u -v google.golang.org/protobuf/cmd/protoc-gen-go \
     google.golang.org/grpc/cmd/protoc-gen-go-grpc \
     github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway \
     github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2 \
     github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc
 ```
+
+* Updated
+
+Current not support go get without module => then switch to command below: For more information, [see](https://golang.org/doc/go-get-install-deprecation)
```

Comment:
@henry0hai Not sure if this line is understandable. Can you just replace the original command directly?